src/Microsoft.ML.Data/DataLoadSave/DataOperationsCatalog.cs

```diff
@@ -168,5 +168,59 @@ public IDataView FilterByMissingValues(IDataView input, params string[] columns)
 
             return new NAFilter(Environment, input, complement: false, columns);
         }
+
+        /// <summary>
+        /// Skip <paramref name="count"/> rows in <paramref name="input"/>.
+        /// </summary>
+        /// <remarks>
+        /// Skips the first <paramref name="count"/> rows from <paramref name="input"/> and returns an <see cref="IDataView"/> with all other rows.
+        /// </remarks>
+        /// <param name="input">The input data.</param>
+        /// <param name="count">Number of rows to skip.</param>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[Skip](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/Skip.cs)]
+        /// ]]>
+        /// </format>
+        /// </example>
+        public IDataView Skip(IDataView input, long count = SkipTakeFilter.Arguments.DefaultSkip)
```

Comment:
>Skip [](start = 25, length = 4)

would  'SkipRows' be more concrete?

Reply:
I really struggled with this one. None of the other Filters mention Rows, so I will leave it off here.

---
In reply to: [254394015](https://github.com/dotnet/machinelearning/pull/2436#discussion_r254394015) [](ancestors = 254394015)